distribution/std-lib/Base/src/Data/Vector.enso

```diff
@@ -397,6 +399,32 @@ type Vector
     rest : Vector | Nothing
     rest = this.tail
 
+    ## Sort the Vector.
```

Comment:
Please make this method PRIVATE for now and clearly write that it does not work.